synapse/config/server.py

```diff
@@ -572,6 +572,7 @@ def read_config(self, config, **kwargs):
             _warn_if_webclient_configured(self.listeners)
 
         self.gc_thresholds = read_gc_thresholds(config.get("gc_thresholds", None))
+        self.gc_seconds = self.read_gc_intervals(config.get("gc_min_interval", None))
```

Comment:
`parse_duration` returns milliseconds, so I think you need to do some renaming and stuff